package.json

```diff
@@ -9,6 +9,8 @@
     "deasync": "^0.1.19",
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
-    "mysql": "^2.18.1"
+    "jquery": "^3.5.0",
+    "mysql": "^2.18.1",
+    "standard": "^14.3.3"
```

Comment:
standard is not dependency, it is not required to execute this project. Theoretically it could be added as `devDependencies`. But unless you can name an advantage this would bring, please remove the dependency completely.